### PR TITLE
fix: Typo in summary and description of `/type/` endpoint

### LINKED ISSUE #43: fix: Typo in endpoint details
https://github.com/tokitou-san/MangaAPI/blob/4a0660358d2e40053921458c2588e0f0af82fdfd/app/api/endpoints.py#L139-L143

Here, used summary and description of `genre` instead `type` ( mistake )
fix: Change `summary` and `description` for `type` endpoint

### PR DESCRIPTION
**Pull request by issamansur**
Closes #43 
Fix typos in endpoint:
- summary: `Genre` -> `Type`
- description: `genre` -> `type`, `ight-novel` -> `light`

### CODE REVIEW

File: app/api/endpoints.py [hunk 1]
```diff
@@ -139,8 +139,8 @@ async def genre(
 @router.get(
     "/type/{type}",
     response_model=list[BaseSearchModel],
-    summary="Genre",
-    description="Search Mangas with genres. eg: `/type/manga/` - returns a list of Mangas with type `manga`. Also has `page` query which get each pages of Mangas ( 1 page contains 18 Mangas ): valid `type` queries - `manga` `one-shot` `doujinshi` `ight-novel` `manhwa` `manhua` `comic`.",
+    summary="Type",
+    description="Search Mangas with types. eg: `/type/manga/` - returns a list of Mangas with type `manga`. Also has `page` query which get each pages of Mangas ( 1 page contains 18 Mangas ): valid `type` queries - `Manga`, `One-shot`, `Doujinshi`, `Light`, `Manhwa`, `Manhua`, `Comic`.",
```
Comment (tokitou-san) [inline app/api/endpoints.py]: Hi @issamansur, thanks for the PR :)
Here can we revert these to lowercases? like `manga` `one-shot` `doujinshi` `light-novel` `manhua` `comic`
cause if I submit endpoint something like this: `/type/Manga` - errorrrrrrr :\

Or convert to lowercase like this
```py
slugified_sort = string_helper.slugify(sort.lower(), "-")
```
I would greatly appreciate if you can do both :)

**Review by tokitou-san — CHANGES_REQUESTED** [review]

**Comment by issamansur** [comment]
Done!

File: app/api/endpoints.py [hunk 2]
```diff
@@ -149,7 +149,8 @@ def type(
     offset: int = 0,
     limit: int = Query(10, le=18),
 ):
+    slugified_type = string_helper.slugify(type, "-")
     slugified_sort = string_helper.slugify(sort, "-")
```
Comment (tokitou-san) [inline app/api/endpoints.py:153]: Yea, one last change
convert to lowercase?

```py
slugified_type = string_helper.slugify(type.lower(), "-")
slugified_sort = string_helper.slugify(sort.lower(), "-")
```

Comment (issamansur) [reply app/api/endpoints.py:153]: But I think `slugify()` do it or did I make a mistake?
```
    def slugify(self, string: str, symbol: str) -> str:
        """Makes lowercase and strip spaces and replace space between words with "+" symbol"""
        string = string.lower().strip()
        string = re.sub(r"[^\w\s-]", "", string)
        string = re.sub(r"[\s_-]+", f"{symbol}", string)
        string = re.sub(r"^-+|-+$", "", string)
        return string
```

**Review by tokitou-san — APPROVED** [review]
Okay LGTM
thanks @issamansur

**Pull request merged**